app/app.go

```diff
@@ -1155,6 +1219,11 @@ func (app *Evmos) setupUpgradeHandlers() {
 		// no store upgrade in v9 or v9.1
 	case v10.UpgradeName:
 		// no store upgrades in v10
+	case v11.UpgradeName:
+		// add ica module in v11
+		storeUpgrades = &storetypes.StoreUpgrades{
+			Added: []string{icatypes.ModuleName},
```

Comment:
Is it only one module that we need to add? shouldn't we add both the host and controller to the store?

This needs to be double checked